ui/pages/settings/security-tab/security-tab.component.js

```diff
@@ -355,6 +357,11 @@ export default class SecurityTab extends PureComponent {
       });
     };
 
+    const handleIpfsToggle = (url) => {
+      url === null || url.length < 1
```

Comment:
```suggestion
      url?.length >=1 ? handleIpfsGatewayChange(‘’) : handleIpfsGatewayChange(IPFS_DEFAULT_GATEWAY_URL)
```

Reply:
What do you think ? :)

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/20172/commits/dfa2f838aeea7be06f3b23cd391a959251225514